test_pmemdEngine/test.py

```diff
@@ -0,0 +1,23 @@
+import htmd
+import shutil
+from glob import glob
+
+# delete previous test
+for directory_i in glob('./test_pmemdEngine/*_test_pmemdEngine'):
+    shutil.rmtree(directory_i)
+
+# run adaptive sampling with MSM
+adapt = htmd.AdaptiveRun()
+adapt.app = htmd.apps.pmemdlocal.PmemdLocal(
+    pmemd_cuda='/usr/local/amber/bin/pmemd.cuda_SPFP')
+adapt.project = 'Test'
+adapt.nmin = 4
+adapt.nmax = 4
+adapt.nepochs = 10
+adapt.inputpath = 'test_pmemdEngine'
+adapt.generatorspath = 'test_pmemdEngine'
```

Comment:
This was an early test script, that we should update. At the time we wrote this we weren't quite sure about the function of the generators and were mainly looking out for debug messages.